packages/p2p/src/components/order-details/order-details.jsx

```diff
@@ -52,10 +59,26 @@ const OrderDetails = observer(({ onPageReturn }) => {
 
     const { chat_channel_url } = sendbird_store;
 
+    const [should_expand_all, setShouldExpandAll] = React.useState(false);
+    const remaining_review_time = React.useRef();
+
+    const page_title = is_buy_order_for_user
+        ? localize('Buy {{offered_currency}} order', { offered_currency: account_currency })
+        : localize('Sell {{offered_currency}} order', { offered_currency: account_currency });
+
+    const rating_average_decimal = review_details ? Number(review_details.rating).toFixed(1) : undefined;
+
     React.useEffect(() => {
         const disposeListeners = sendbird_store.registerEventListeners();
         const disposeReactions = sendbird_store.registerMobXReactions();
 
+        order_store.setRatingValue(0);
+        order_store.setIsRecommended(undefined);
```

Comment:
`nit` Is it safe to use `undefined` here? I remembered Yashim told us to use `null` instead